backend/src/eml/tests/deserialize_eml510b_test.eml.xml

```diff
@@ -1,15 +1,14 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why was the XML prolog removed?

Reply:
Me being clunky ;) Re-added in [1978245](https://github.com/kiesraad/abacus/pull/637/commits/1978245d5c958badcd213c70abee535ed5f5e0a3)